functions/components/metaTagInjector.ts

```diff
@@ -7,44 +7,27 @@ type MetaTagInjectorInput = {
 export class MetaTagInjector {
```

Comment:
The `MetaTagInjectorInput` properties should not use `any`, it should require `string`s.